### ISSUE TITLE
[4.0.0+] Incorrect TS definition for the `setDataAtCell` method

### ISSUE THREAD

**Issue by mrpiotr-dev** [handsontable/handsontable#8601]
### Description
Our TS definition for `setDataAtCell` suggests that `col` might be `string` or `number` - however, in the source code, we have the following condition:
```
if (typeof input[i][1] !== 'number') {
        throw new Error('Method `setDataAtCell` accepts row and column number as its parameters. If you want to use object property name, use method `setDataAtRowProp`'); // eslint-disable-line max-len
      }
```

### Steps to reproduce
1. Initialize Handsontable
2. Run `setDataAtCell(0, '0', 'new value')`
You will get an error.

### Demo
https://jsfiddle.net/fovqhg47/ (check dev-tools' console)

### Your environment
* Handsontable version: >=4.0.0
* Browser Name and version: any
* Operating System: any

Inform Zen#26759

**Comment by aninde** [comment]
This issue will be fixed on the upcoming v11.1.0 https://jsfiddle.net/jx7nezm1/.  As the author mentioned 

> The second argument (column index) has to be passed as a number. Otherwise, the exception is thrown.

**Comment by AMBudnik** [comment]
Issue closed as fixed for v 11.1.0
